src/ontology/mondo-edit.obo

```diff
@@ -258297,6 +258323,7 @@ name: autosomal recessive limb-girdle muscular dystrophy type 2O
 def: "Autosomal recessive limb-girdle muscular dystrophy type 2O (LGMD2O) is a form of limb-girdle muscular dystrophy characterized by an onset in childhood or adolescence of rapidly progressive proximal limb muscle weakness (particularly affecting the neck, hip girdle, and shoulder abductors), hypertrophy in the calves and quadriceps, ankle contractures, and myopia." [Orphanet:206564]
 subset: ordo_disease {source="Orphanet:206564"}
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in POMGNT1" EXACT [MONDO:design_pattern]
+synonym: "LGMD-POMGNT1 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/